acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -6,11 +6,12 @@ module Puppet
   module Acceptance
     module InstallUtils
       PLATFORM_PATTERNS = {
-        :redhat        => /fedora|el|centos/,
         :debian        => /debian|ubuntu/,
         :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
+        :redhat        => /fedora|el|centos/,
+        :sles          => /sles/,
         :solaris       => /solaris/,
-        :windows       => /windows/,
+        :windows       => /windows/
```

Comment:
Was this for style reasons or was there breakage? 

FWIW, I always like the last element to have a trailing `,` (when the language permits it) so that future additions have a cleaner diff. But that's strictly preference.